can_tools/scrapers/official/ID/id_county.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import us
+
+from can_tools.scrapers.official.base import TableauDashboard
+from can_tools.scrapers import variables
+class IdahoCountyVaccine(TableauDashboard):
+    has_location = False
+    location_type = "county"
+    source = "https://coronavirus.idaho.gov/"
+    source_name = "Idaho Official Government Website"
+    data_tableau_table = 'Vax Rate / County Chart'
+    baseurl = "https://public.tableau.com"
+    viewPath = "COVID-19VaccineDataDashboard/VaccineUptake"
+
+    state_fips = int(us.states.lookup("Idaho").fips)
+
+    variables = {
+        '1': variables.INITIATING_VACCINATIONS_ALL,
+        '2': variables.FULLY_VACCINATED_ALL
+    }
+
+    
+    def normalize(self, data):
+        data.columns = [
```

Comment:
This looks scary to me, is there a way we can rename existing columns?

We've seen so many instances of columns being re-ordered or renamed that I hesitate to assume constant ordering

Reply:
Changed